src/app/styl/views/main_window.styl

```diff
@@ -1,8 +1,16 @@
 html,
 body,
+    text-rendering: optimizelegibility
+    -webkit-font-smoothing: subpixel-antialiased;
+    color: #FAFAFA
```

Comment:
this is hardcoded color. it's probably not a good idea since we work with themes for the colors